requirements.txt

```diff
@@ -194,6 +194,7 @@ python-dateutil==2.8.1
     #   pandas
     #   searchtweets-v2
     #   zenpy
+python-facebook-api==0.9.2
```

Comment:
need to add this in requirement.in and then we need to generate it via following command -
```
pip-compile requirements.in > requirements.txt
```

Also please add dependency for conda as well for conda folder